examples/source/1.components/amp-story-interactive-quiz.html

```diff
@@ -218,6 +222,10 @@
 </html>
 
 <!--
+  ## Accessibility considerations
+
+  Currently, this component lacks keyboard and screen reader accessibility.
```

Comment:
Same as above (link to https://github.com/ampproject/amphtml/issues/32751)

Reply:
Currently working on the PR to make it accessible, so no need to add it to the documentation